index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
```

Comment:
- [ **OPTIONAL** ] I think it would be better :-1:  if you rename your **page title** to a meaningful name to help users have an idea about your page even before the access and for best practices. I suggest the name of **the project** you are doing.